tlslite/messages.py

```diff
@@ -1765,3 +1759,61 @@ def parse(self, p):
 
     def write(self):
         return self.bytes
+
+
+class Heartbeat(object):
+    """
+    Handling Heartbeat extension from RFC 6520
+
+    @type message_type: int
+    @ivar message_type: type of message (response or request)
+
+    @type payload_length: int
+    @ivar payload_length: length of sending payload
+
+    @type payload: bytes
+    @ivar payload: payload
+    """
+    def __init__(self):
+        self.contentType = ContentType.heartbeat
+        self.message_type = 0
+        self.payload = bytearray(0)
+        self.padding = bytearray(0)
+
+    def create(self, message_type, payload, padding_length):
+        """Create heartbeat request or response with selected parameters"""
+        self.message_type = message_type
+        self.payload = payload
+        if not(15 < padding_length < 2**14 - len(self.payload) - 3):
```

Comment:
for one, `not` is not a function, it's a reserved keyword, like `if`, `else`, `or`, etc. so there needs to be a space between it and `(`

I have checked if `not (_ < variable <_)` behaves the same as `not _ < variable < _` and they seemed to be exactly equivalent, but maybe I missed some case...

but true, all of it is moot given that this test shouldn't be here in the first place